app/views/admin/editions/_review_reminder_fields.html.erb

```diff
@@ -0,0 +1,55 @@
+<div class="govuk-!-margin-bottom-8">
+  <%= form.fields_for :document do |document_form| %>
+    <%= document_form.fields_for :review_reminder do |review_reminder_form| %>
+      <%= hidden_field_tag "edition[document_attributes][review_reminder_attributes][creator_id]", current_user.id %>
```

Comment:
yea makes sense. I think it's probs easiest to handle it as a before action rather than worrying about doing a deep_merge 

```
  before_action :set_creator_for_review_reminder, only: %i[create update]

  def set_creator_for_review_reminder
    edition_params["document_attributes"]["review_reminder_attributes"]["creator_id"] = current_user.id
  end
```

does that seem okay?